frontend/src/store/slices/UserSlice.ts

```diff
@@ -2,51 +2,51 @@ import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
 import type { RootState } from "store/store";
 
 export interface User {
-	username: string;
-	loading: boolean;
-	error: string | undefined;
+    username: string;
+    loading: boolean;
+    error: string | undefined;
 }
 
 const initialState: User = {
-	username: "",
-	loading: false,
-	error: ""
+    username: "",
+    loading: false,
+    error: ""
 };
 
-export const fetchUser = createAsyncThunk("user/fetchUser", async thunkAPI => {
-	const getRandomArbitrary = (min: number, max: number) => {
-		return Math.ceil(Math.random() * (max - min) + min);
-	};
-	const response = await fetch(
-		`https://dummyjson.com/users/${getRandomArbitrary(30, 1)}`,
-		{
-			method: "GET"
-		}
-	);
-	return response.json();
+export const fetchUser = createAsyncThunk("user/fetchUser", async () => {
+    const getRandomArbitrary = (min: number, max: number) => {
+        return Math.ceil(Math.random() * (max - min) + min);
+    };
+    const response = await fetch(
+        `https://dummyjson.com/users/${getRandomArbitrary(30, 1)}`,
+        {
+            method: "GET"
+        }
```

Comment:
I think method is redundant

Reply:
That's not my code, not from that PR, but that's true, fixed now :)